src/App.js

```diff
@@ -1,43 +1,16 @@
-import "./App.css";
-import logo from "./logo.png";
+import "./components/navbar/navbar.css";
+// import {LandingPage} from "./pages/landing-page.jsx"
 
+import { Navbar } from "./components/navbar/navbar.js";
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+    <div>
+      <img id="bg-img" src="https://i.ibb.co/bQHd3Sd/intro-bg-1.jpg" alt="" />
```

Comment:
create separate home component and put the banner image there.

Reply:
Yupp